tests/threshold/threshold-config-threshold-limit-flow/threshold.config

```diff
@@ -0,0 +1 @@
+threshold gen_id 1, sig_id 1000001, type limit, track by_flow, count 1, seconds 60
```

Comment:
Could we test another value for count ? Like 10 out of the 19

Reply:
done in another test now

Reply:
Seen in tests/threshold/threshold-rule-flow/test.rules ok